images/ubuntu2004/scripts/postinstall.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+echo "ubuntu" | sudo -S cp /etc/apt/sources.list /etc/apt/sources.list.bak
+
+sudo sed -i 's/http:\/\/archive\.ubuntu\.com\/ubuntu\//http:\/\/azure\.archive\.ubuntu\.com\/ubuntu\//g' /etc/apt/sources.list
+sudo sed -i 's/http:\/\/[a-z][a-z]\.archive\.ubuntu\.com\/ubuntu\//http:\/\/azure\.archive\.ubuntu\.com\/ubuntu\//g' /etc/apt/sources.list
+sudo apt-get update
+
+sudo apt update
+sudo apt install linux-azure linux-image-azure linux-headers-azure linux-tools-common linux-cloud-tools-common linux-tools-azure linux-cloud-tools-azure
```

Comment:
nit: `linux-azure` is already installed by this point

Why not install all of these in cloud-init?

Reply:
Thanks, moved everything to install via cloud-init